About.html

```diff
@@ -0,0 +1,117 @@
+          <div class="button-signup">
+            <button class="signup">
+            Sign Up
+            </button>
+          </div>
+        </div>
+        <!-- Mobile-menu -->
+        <div id="about-menu">
+          <a id="about-exit">&times;</a>
+          <ul id="about-items">
+            <li class="link"><a href="index.html">Home</a>
+            </li>
+            <li class="link" ><a href="#logo">Logo</a>
+            </li>
+            <li class="link"><a href="#events">Events</a>
+            </li>
+            <li class="link"><a href="#footer">Footer</a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+    <!--About-->
+    <section class="about" id="about">
+      <p class="about-text">"Be agents of change"</p>
+      <h1 class="about-heading">
+        SCHOOL OF COMMUNITY LEADERSHIP
+      </h1>
+      <div class="paragraph">
+        <p>
+          School of community leadership is a leadership development programme for professionals from all fields.
+          It is aimed at equipping leaders to grow into agents of change and effectively bring about positive transformation in their family, workplace,community and nation.
+        </p>
+      </div>
+      <p class="contact">
+        Please contact us via email for more details and clarifications about socl<br>
+        <b>school.leadership@socl.com</b>
+      </p>
+    </section>
+  </div>
+  
+  <!--Logo-->
+  <section class="about-logo" id="logo">
+    <h2 class="logo-heading">
+      SOCL 
+    </h2>
+    <p class="logo-message">
+      Equipping leaders to be agents of positive transformation!
+    </p>
+    <div class="logo-image">
+      <img class="logo" src="images/logo3.png" alt="logo">
+    </div>
+  </section>
+  
+  <!--Past events-->
+  <section class="past-events" id="events">
+    <h3 class="events-heading">
+      See the past participants message
+    </h3>
+    <p class="event-message">
+      Lorem ipsum dolor sit amet consectetur, adipisicing elit. Earum delectus necessitatibus quo!
+    </p>
+    <div class="participants">
+      <div class="participant1 card text-white">
+        <img src="images/event2.jpg" class="card-img" alt="event1">
+        <div class="card-img-overlay">
+          <h3 class="card-title">2020</h3>
+          <p class="card-text">Lorem ipsum dolor sit.</p>
+        </div>
+      </div>
+      <div class="participant2 card text-white">
+        <img src="images/replace.jpg" class="card-img" alt="event2">
+        <div class="card-img-overlay">
+          <h3 class="card-title">2021</h3>
+          <p class="card-text">Lorem ipsum dolor sit.</p>
+        </div>
+      </div>
+    </div>
+  </section>
+  <footer id="footer">
+    <div class="footer-image">
+      <img class="footer-logo" src="images/logo3.png" alt="logo">
+    </div>
+    <div class="info">
+      <p>@socl School of Community Leadership</p>
+      <p class="sub"><sub>Some rights are reserved</sub></p>
+    </div>
+  </footer>
```

Comment:
- Kindly try to match the footer with the Behance design in the mobile version:
   - Kindly change the ` background-color ` to be white.
   - Kindly make the logo and paragraph centered vertically in the container.
   - Kindly change the `font-size` of the paragraph to be smaller.
- Kindly Kindly make the logo and paragraph centered vertically in the container in the desktop version.

| Your design | Behance design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/157788998-656c17ab-55c6-4e28-80ae-1c93394d8473.png) | ![image](https://user-images.githubusercontent.com/24703171/157789076-4465df0a-7104-46f2-967c-3879aa504231.png) |

Reply:
Addressed!